Meter.c

```diff
@@ -124,26 +125,48 @@ static void BarMeterMode_draw(Meter* this, int x, int y, int w) {
 
    // First draw in the bar[] buffer...
    int offset = 0;
+   double actualBarWidth=0;
    for (uint8_t i = 0; i < this->curItems; i++) {
       double value = this->values[i];
       if (isPositive(value) && this->total > 0.0) {
          value = MINIMUM(value, this->total);
-         blockSizes[i] = ceil((value / this->total) * w);
+         actualBarWidth = ((value / this->total) * w);
+         blockSizes[i] = ceil(actualBarWidth);
       } else {
          blockSizes[i] = 0;
       }
       int nextOffset = offset + blockSizes[i];
       // (Control against invalid values)
       nextOffset = CLAMP(nextOffset, 0, w);
+
+      const Settings* settings = this->host->settings;
+      static const wchar_t* bars[8] = {
+         L" ||||||||",
+         L" ########",
+         L"⠀⡀⡄⡆⡇⣇⣧⣷⣿",
+         L" ░░▒▒▓▓██",
+         L" ▏▎▍▌▋▊▉█",
+         L" ▁▂▃▄▅▆▇█",
+         L" ▌▌▌▌████",
+         L" ▔🮂🮃▀🮄🮅🮆█"
+      };
+
       for (int j = offset; j < nextOffset; j++)
```

Comment:
Can you add braces for this `for` loop? TIA.

Reply:
will do that :D

Reply:
done :D